server/rosetta/converter_test.go

```diff
@@ -0,0 +1,355 @@
+package rosetta_test
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/server/rosetta"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+
+	rosettatypes "github.com/coinbase/rosetta-sdk-go/types"
+	"github.com/stretchr/testify/suite"
+	crgerrs "github.com/tendermint/cosmos-rosetta-gateway/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtx "github.com/cosmos/cosmos-sdk/x/auth/tx"
+	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+type ConverterTestSuite struct {
+	suite.Suite
+
+	c               rosetta.Converter
+	unsignedTxBytes []byte
+	unsignedTx      authsigning.Tx
+
+	util struct {
+		ir     codectypes.InterfaceRegistry
+		cdc    *codec.ProtoCodec
+		txConf client.TxConfig
+	}
```

Comment:
the `util` usually should be avoided. How about moving the `util` struct fields up (directly into ConverterTestSuite)?

Reply:
sounds good to me